src/approximate_inference.jl

```diff
@@ -0,0 +1,32 @@
+import Base: rand, length
+import Distributions: logpdf, AbstractMvNormal
+
+export elbo, dtc
+export SparseFiniteGP
+
+struct SparseFiniteGP{T1<:FiniteGP, T2<:FiniteGP} <: AbstractMvNormal
```

Comment:
We probably need a docstring here. Would be good to include a doctest with example usage + explain that _all_ that this abstraction currently does is replace the implementation of `logpdf` with a call to `elbo`.

Reply:
Done. Let me know if you've got other ideas for doctests to include.